docs/app/pages/css/css-sections/icons/ux-icons/ux-icons.component.html

```diff
@@ -1,5 +1,18 @@
-<p *ngIf="iconSetClass === 'hpe-icon'">To use an icon simply apply the <code>{{ iconSetClass }}</code>
-    class along with one of the following classes:</p>
+<div *ngIf="iconSetClass === 'hpe-icon'" class="row m-b-md hpe-icon-instructions">
+
+    <div class="col-lg-12 col-md-12 col-sm-12">
+
+        <p>To use an <code>{{ iconSetClass }}</code> import the HPE Icons CSS file into your project.
+            It can be found in the <code>@ux-aspects/ux-aspects</code> package:</p>
+
+        <uxd-snippet language="html" [code]="hpeIconsLinkHtml"></uxd-snippet>
```

Comment:
Is this the correct way to show this snippet of html?

Reply:
You can remove the `language` attribute now and use `[content]` instead of `[code]`. You need to create a snippets folder and put the code sample file in there. If you look in other documentation sections we add this line into the constructor:

```typescript
super(require.context('./snippets/', false, /(html|css|js|ts)$/));
```

Which precompiles everything in the snippets folder and makes them available in the component on a `snippets` object. You can either access the raw string or the preformatted code snippet, eg, if the file is called `app.ts`, then you can access it by doing:

```html
[content]="snippets.compiled.appTs"
```